extension/intents/clipboard/clipboard.js

```diff
@@ -2,7 +2,7 @@ import * as intentRunner from "../../background/intentRunner.js";
 import * as content from "../../background/content.js";
 import * as browserUtil from "../../browserUtil.js";
 
-async function copy(context, copyType, complete = false) {
+async function copy(context, copyType, complete = false, value) {
```

Comment:
Huh, I'm not even entirely sure what a default value in the not-last position does. It would look clearer to me if `value` had a default value too. Or just change the third argument to an options object, since it's hard to keep track of positional arguments with that many.